### PR TITLE
Update macos.markdown

### PR DESCRIPTION
**Pull request by mbierman**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/installation/macos.markdown [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "Installation on macOS"
 description: "Installation of Home Assistant on your macOS system."
 ---
 
-[macOS](http://www.apple.com/macos/) is available by default on Apple computer. If you run a different operating system, please refer to the other section of the documentation.
+[macOS](http://www.apple.com/macos/) is available by default on macOS. If you run a different operating system, please refer to the other section of the documentation.
```
Comment (fabaff) [inline source/_docs/installation/macos.markdown:6]: This change doesn't make sense to me.

**Comment by mbierman** [comment]
Wouldn't this  also run on hackintoshes which aren't made by Apple? In other words, the hardware isn't really relevant is it? the OS is what is important.

I suppose you could say, "is available by default on Apple computer**s**" But clearly it doesn't run on iPads.

**Comment by frenck** [comment]
@mbierman It get what you are trying to say, however,
"macOS is available by default on macOS"

Is the same as:

"Linux is available by default on Linux" and "Windows is available by default on Windows"...

**Comment by mbierman** [comment]
> @mbierman It get what you are trying to say, however,
> "macOS is available by default on macOS"
> 
> Is the same as:
> 
> "Linux is available by default on Linux" and "Windows is available by default on Windows"...

Agreed. I would drop that whole sentence if it were me. 

On a different note, is there some step beyond what is on this page to get things running?

**Comment by frenck** [comment]
Giving it some more thought. While there is a thing called Hackingtosh, it is not supported by Apple in general. Hence, we should not support it as well.

I think the current documentation makes sense. macOS is delivered on an Apple computer.

If one has a different, unsupported, set up, that is fine. But not up to Apple, nor us.

Nevertheless, thank you for contributing! That is always appreciated. 👍

**Pull request closed without merging**